extensions/mock-credentials-verifier/src/main/java/org/eclipse/dataspaceconnector/mvd/MockCredentialsVerifier.java

```diff
@@ -66,9 +73,18 @@ public Result<Map<String, Object>> verifyCredentials(String hubBaseUrl, PublicKe
                             s -> (Object) decode(s[1], UTF_8)
                     ));
             monitor.debug("Completing (mock) credential verification. Claims: " + claims);
-            return Result.success(claims);
+            return Result.success(toMappedVerifiableCredentials(claims));
         } catch (MalformedURLException e) {
             throw new EdcException(e);
         }
     }
+
+    private Map<String, Object> toMappedVerifiableCredentials(Map<String, Object> regionClaims) {
```

Comment:
This code is duplicated in the tests, but this class MockCredentialsVerifier will be deleted in the next PR so no action needed.